dotCMS/src/main/java/com/dotcms/contenttype/model/field/FieldType.java

```diff
@@ -0,0 +1,75 @@
+package com.dotcms.contenttype.model.field;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collection;
+import java.util.Map;
+import static com.dotcms.util.CollectionsUtils.map;
+
+/**
+ * Intances of the class FieldType represent a Field's Type.
+ * it provide Field's Type meta data.
+ */
+public class FieldType implements Comparable<FieldType>{
+    private String id;
+    private String label;
+    private Collection<ContentTypeFieldProperties> properties;
+    private String helpText;
+    private String clazz;
+
+    public FieldType(String id, String label, Collection<ContentTypeFieldProperties> properties, String helpText, String clazz) {
+        this.id = id;
+        this.label = label;
+        this.properties = properties;
+        this.helpText = helpText;
+        this.clazz = clazz;
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public String getLabel() {
+        return label;
+    }
+
+    public Collection<ContentTypeFieldProperties> getProperties() {
+        return properties;
+    }
+
+    public String getHelpText() {
+        return helpText;
+    }
+
+    public Map<String, Object> toMap(){
+        return map("id", id,
+                "label", label,
+                "properties", properties,
+                "helpText", helpText);
+    }
+
+    public String getClazz() {
+        return clazz;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Missing curly brace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00122)